packages/web-components/src/components/va-banner/test/va-banner.e2e.ts

```diff
@@ -33,6 +33,41 @@ describe('va-banner', () => {
     `);
   });
 
+  it('adds sr-only span to headline when headline prop is provided', async () => {
+    const page = await newE2EPage();
+    await page.setContent(
+      '<va-banner headline="Important announcement"></va-banner>',
+    );
+
+    await page.waitForChanges();
+
+    // Find the h3 headline element inside va-alert
+    const headline = await page.find('va-banner >>> va-alert h3[slot="headline"]');
+    expect(headline).not.toBeNull();
+
+    // Check that sr-only span was added
+    const srOnlySpan = await page.find('va-banner >>> va-alert h3[slot="headline"] .usa-sr-only');
+    expect(srOnlySpan).not.toBeNull();
+
+    const srOnlyText = await srOnlySpan.textContent;
+    expect(srOnlyText).toBe('Information Alert ');
```

Comment:
Do we want `va-banner` (which uses va-alert internally) to have different sr-only text? It only uses the information variation of va-alert so it currently announces that way.

<img width="772" height="446" alt="Image" src="https://github.com/user-attachments/assets/77afe13f-48a8-4ade-b1cb-aad7bf5d691e" />